environment.yml

```diff
@@ -2,38 +2,158 @@ name: skorch
 channels:
   - defaults
   - pytorch
-  - conda-forge
 dependencies:
   - _libgcc_mutex=0.1
+  - absl-py=0.12.0
+  - aiohttp=3.7.4
+  - alabaster=0.7.12
+  - astroid=2.5
+  - async-timeout=3.0.1
+  - attrs=20.3.0
```

Comment:
For `environment.yml`, I have been starting to prefer the following for dev environments:

```bash
conda env export -n skorch --from-history
```

Reply:
I didn't know about that option, I will try it out and update the environment.yml in case it works, thanks!

Reply:
Okay, it worked. It's still quite cluttered but a bit better and I didn't need to clean up as much. What I did need to do: Add non default channels, add the `pip:` section, which is somehow omitted. Good news is that even including the dev deps, only sacred needs to be installed via pip.